### DISCUSSION TITLE
[V5] How to setup VSCode debug for adonis?

### DISCUSSION THREAD

**Discussion by janainascal** [adonisjs/core#2375] [Help]
# Previous config
In found these [blog](https://viglucci.io/debugging-adonis-with-vscode) teatching  how to setup debug, but, in that version, the server had a debug option:
```
adonis serve --dev --debug
``` 

In another discussion I found [this debug setup](https://github.com/adonisjs/core/discussions/1343#discussioncomment-253871), but  the debbuger trhrow a error:
```
AdonisJS expects ".adonisrc.json" file to exist in the application root
```
So my question is, How to setup debug on vscode for adonis version 5?

# Versions
node: v14.15.4
npm: 6.14.10
dependencies:
    "typescript": "^4.1.5",
    "@adonisjs/auth": "^5.1.1",
    "@adonisjs/core": "^5.0.5-canary-rc-2",
    "@adonisjs/lucid": "^10.0.0",
    "@adonisjs/repl": "^1.1.6",
    "luxon": "^1.25.0",
    "@adonisjs/assembler": "^4.0.1",
    "@openapitools/openapi-generator-cli": "^2.2.1",
    "@types/jsdom": "^16.2.6",
    "@types/supertest": "^2.0.10",
    "adonis-preset-ts": "^2.0.0",
    "execa": "^5.0.0",
    "get-port": "^5.1.1",
    "japa": "^3.1.1",
    "jsdom": "^16.5.0",
    "pino-pretty": "^4.5.0",
    "supertest": "^6.1.3",
    "youch": "^2.1.1",
    "youch-terminal": "^1.0.1"
    "marked": "^2.0.1",
    "pg": "^8.5.1",
    "phc-argon2": "^1.1.0",
    "proxy-addr": "^2.0.6",
    "reflect-metadata": "^0.1.13",
    "source-map-support": "^0.5.19"

**Comment by satheler** [comment]
Add this in your `.vscode/launch.json`

```json
{
  "version": "0.2.0",
  "configurations": [
    {
      "type": "node",
      "request": "launch",
      "name": "Launch Server",
      "skipFiles": [
        "<node_internals>/**"
      ],
      "program": "${workspaceFolder}/ace",
      "args": ["serve", "--watch"],
      "outFiles": [
        "${workspaceFolder}/build/**/*.js"
      ]
    }
  ]
}
```
